pkg/importer/spanner/sysl.arrai

```diff
@@ -0,0 +1,138 @@
+# size returns the size of an attribute
+let size = \length
+    cond {
+        length = 'MAX': '',
+        //seq.has_prefix('0x', length): '(' ++ //seq.trim_prefix('0x', length) ++ ')',
+        length > 0: $`(${length})`,
+        _: '',
+    };
+
+# attributePatternToken returns the token based on attribute value
```

Comment:
Nit: as a reader, this is a bit confusing. What is a token? Why would you just want to return the same argument you passed in in one of four cases? Same below.

Reply:
attributePatternToken checks if the attr passed is included in a foreign key and return a token if included. Token is generally a pattern delimiter which needs to be rendered conditionally.